configs/schema/configJsonSchema.js

```diff
@@ -359,6 +363,54 @@ module.exports = {
                   additionalProperties: false,
                   required: ['createEmptyBlocks', 'createEmptyBlocksInterval'],
                 },
+                log: {
+                  type: 'object',
+                  properties: {
+                    level: {
+                      type: 'object',
+                      properties: {
+                        'abci-client': {
+                          $ref: '#/definitions/tenderdashLogModule',
+                        },
+                        blockchain: {
+                          $ref: '#/definitions/tenderdashLogModule',
+                        },
+                        consensus: {
+                          $ref: '#/definitions/tenderdashLogModule',
+                        },
+                        main: {
+                          $ref: '#/definitions/tenderdashLogModule',
+                        },
+                        mempool: {
+                          $ref: '#/definitions/tenderdashLogModule',
+                        },
+                        p2p: {
+                          $ref: '#/definitions/tenderdashLogModule',
+                        },
+                        'rpc-server': {
+                          $ref: '#/definitions/tenderdashLogModule',
+                        },
+                        state: {
+                          $ref: '#/definitions/tenderdashLogModule',
+                        },
+                        statesync: {
+                          $ref: '#/definitions/tenderdashLogModule',
+                        },
+                        '*': {
+                          $ref: '#/definitions/tenderdashLogModule',
+                        },
+                      },
+                      required: ['*'],
```

Comment:
Does the wildcard is required? Or `log_level` can't be empty string?

Reply:
Yes, some minimum `log_level` must be defined, but it doesn't have to be `'*'`, we could set `main` instead. If it is set blank then the Tenderdash container restarts with error `ERROR: empty log level`.

Reply:
Possible log levels are listed here: https://docs.tendermint.com/master/nodes/logging.html#list-of-modules
I'm not sure what `main` is actually, it is not in the list but is accepted by Tenderdash.

Reply:
I see. Then you can use just `minProperties: 1` instead of `required`